src/AdaptiveCardContainer.tsx

```diff
@@ -164,12 +164,16 @@ class AdaptiveCardContainer extends React.Component<Props, State> {
 
         adaptiveCard.hostConfig = this.props.hostConfig || defaultHostConfig;
 
-        let errors: IValidationError[] = [];
+        let errors: Array<IValidationError|Error> = [];
 
         if (!this.props.nativeCard && this.props.jsonCard) {
-            this.props.jsonCard.version = this.props.jsonCard.version || '0.5';
-            adaptiveCard.parse(cardWithoutHttpActions(this.props.jsonCard));
-            errors = adaptiveCard.validate();
+            try {
+                this.props.jsonCard.version = this.props.jsonCard.version || '0.5';
+                adaptiveCard.parse(cardWithoutHttpActions(this.props.jsonCard));
+                errors = adaptiveCard.validate();
+            } catch (error) {
+                errors.push(error);
```

Comment:
Can you verify how `errors` array get displayed on the screen?

Think about the upstream and downstream when changing the code:
- Downstream
   - Can the screen display `Error`?
   - Can it display `IValidationError` and `Error` at the same time?
   - What's the code over there? There is a command at MockBot v4 to force `IValidationError`, The command is called `card broken` and `card broken:1`
   - Also add a case to MockBot to make sure we capture this regression later
      - MockBot might already have this captured or not, I am not quite sure
- Upstream: there is no upstreamer in this case, so it's good

Also, try it in v4 by visiting https://webchat-playground.azure-websites.net/. If there is a bug over there, file another issue.